tasks/backfill_test_instances.py

```diff
@@ -17,27 +18,34 @@ def run_impl(self, *args, dry_run=True, **kwargs):
             "Updating test instances",
         )
 
-        test_instance_filter = TestInstance.objects.select_related(
-            "upload__report__commit"
-        ).filter(
-            branch=None,
-            commitid=None,
+        test_instance_list = (
+            TestInstance.objects.select_related("upload__report__commit")
+            .filter(
+                Q(branch__isnull=True)
+                | Q(commitid__isnull=True)
+                | Q(repoid__isnull=True)
+            )
+            .all()
         )
-        num_test_instances = test_instance_filter.count()
-        all_test_instances = test_instance_filter.all()
 
-        chunk_size = 1000
-
-        chunks = [
-            all_test_instances[i : i + chunk_size]
-            for i in range(0, num_test_instances, chunk_size)
-        ]
-
-        for chunk in chunks:
-            for test_instance in chunk:
+        for i in range(0, test_instance_list.count(), 1000):
+            updates = []
+            thing = (
```

Comment:
Could we rename this to something more meaningful 😅, seems to me this is something like "test_instances_without_identifiers" or something like that?

Reply:
oops 😅, `test_instances_without_identifiers` sounds good